projects/my-angular-crasher/src/app/app.component.ts

```diff
@@ -48,7 +48,8 @@ export class AppComponent implements OnInit {
     this.link$ = bugsplat.getObservable()
       .pipe(
         map((bugSplatEvent) => {
-          const crashId = bugSplatEvent.responseData.crash_id;
+          // TODO BG test
+          const crashId = bugSplatEvent.responseData.crashId;
```

Comment:
TODO BG

Reply:
Resolved in [12cca6a](https://github.com/BugSplat-Git/bugsplat-ng/pull/93/commits/12cca6acda73de0ff3c4e8f460150247305fb8a4)